ci/gpu/build.sh

```diff
@@ -94,9 +74,11 @@ conda list --show-channel-urls
 # BUILD - Build dask-cuda
 ################################################################################
 
-gpuci_logger "Build dask-cuda"
-cd "$WORKSPACE"
-python -m pip install -e .
+gpuci_logger "Build and install dask-cuda"
+cd "${WORKSPACE}"
+gpuci_conda_retry build conda/recipes/dask-cuda --python="${PYTHON}"
+gpuci_mamba_retry install dask-cuda
```

Comment:
this needs a `-c` flag to ensure that `dask-cuda` is installed from the local filesystem

Reply:
PR updated